addons/hr_holidays/static/src/leave_stats/leave_stats.xml

```diff
@@ -1,30 +1,36 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <templates xml:space="preserve">
     <div t-name="hr_holidays.LeaveStatsComponent" class="o_leave_stats">
-        <div t-if="employee" id="o_leave_stats_employee">
-            <div class="o_hr_leave_subtitle">
-                <t t-esc="employee[1]"/> in <t t-esc="thisYear"/>
+        <t t-if="state.multi_employee or state.holiday_type in ['company', 'category']" id="o_leave_stats_multi_employee">
+            <div class="fst-italic">
+                Information for multiple employees cannot be displayed
             </div>
-            <div t-if="state.leaves.length === 0">
-                None
+        </t>
+        <t t-else="">
+            <div t-if="state.employee" id="o_leave_stats_employee">
+                <div class="o_hr_leave_subtitle">
+                    <t t-esc="state.employee[1]"/> in <t t-esc="thisYear"/>
+                </div>
+                <div t-if="state.leaves.length === 0">
+                    None
+                </div>
+                <div t-foreach="state.leaves" t-as="leave" t-key="leave_index" class="d-flex flex-row justify-content-between">
+                    <span t-esc="leave.holiday_status_id[1]"/>
+                    <span><t t-esc="leave.number_of_days"/> day(s)</span>
+                </div>
             </div>
-            <div t-foreach="state.leaves" t-as="leave" t-key="leave_index" class="d-flex flex-row justify-content-between">
-                <span t-esc="leave.holiday_status_id[1]"/>
-                <span><t t-esc="leave.number_of_days"/> day(s)</span>
+            <div t-if="state.department" id="o_leave_stats_department">
+                <div class="o_horizontal_separator o_hr_leave_subtitle">
+                    <t t-esc="state.department[1]"/>
+                </div>
+                <div t-if="state.departmentLeaves.length === 0">
+                    None
+                </div>
+                <div t-foreach="state.departmentLeaves" t-as="leave" t-key="leave_index" t-attf-class="d-flex flex-row justify-content-between {{leave.sameEmployee ? 'fw-bold': ''}}">
+                    <span><t t-esc="leave.employee_id[1]"/>: <t t-esc="leave.number_of_days"/> day(s)</span>
+                    <span><t t-esc="leave.dateFrom"/> - <t t-esc="leave.dateTo"/></span>
+                </div>
             </div>
```

Comment:
As discussed previously as well, instead of 

```xml
<t t-else="">
    <div t-if="state.employee">
        xxx
    </div>
    <div t-if="state.department">
        xxx
    </div>
</t>
```
We should rather have

```xml
<div t-elif="state.employee">
    xxx
</div>
<div t-elif="state.department">
    xxx
</div>
```
There is no point of having empty else, it increases indentation for no reason.. 


also these conditions are mutually exclusive, does not make sense to have `if`. When you have if, even if the condition `state.employee` is correct, you will still check next condition `state.department`. Which is unnecessary. Moreover, having if, where it should be elif instead, is confusing.